### ISSUE TITLE
Ephemera Folders should have ARKs generated.

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#565]
When?

**Comment by escowles** [comment]
When they are complete, to work the same way as ScannedResources?

**Comment by tpendragon** [comment]
Yeah, but they don't work the same since they're visible when the parent box is marked appropriately

**Comment by hackmastera** [comment]
Workflow has changed; this is now actionable.

**Comment by tpendragon** [comment]
I'm...leaving this open, but there are SO many ephemera items in DPUL now. It wasn't necessary to give them ARKs. However, these don't currently have any persistent identifier because of that.
